src/loki/LokiBase.h

```diff
@@ -43,34 +55,21 @@ class LokiBase : public Detector {
   /** Shared between input_thread and processing_thread*/
   CircularFifo<unsigned int, EthernetBufferMaxEntries> InputFifo;
   /// \todo the number 11 is a workaround
-  RingBuffer<EthernetBufferSize> EthernetRingbuffer{EthernetBufferMaxEntries + 11};
-
-  struct {
-    // Input Counters - accessed in input thread
-    int64_t RxPackets;
-    int64_t RxBytes;
-    int64_t FifoPushErrors;
-    int64_t PaddingFor64ByteAlignment[5]; // cppcheck-suppress unusedStructMember
+  RingBuffer<EthernetBufferSize> RxRingbuffer{EthernetBufferMaxEntries + 11};
 
-    // Processing Counters - accessed in processing thread
-    int64_t FifoSeqErrors;
-    int64_t ReadoutsErrorBytes;
-    int64_t ReadoutsCount;
-
-    int64_t RxIdle;
-    int64_t Events;
-    int64_t EventsUdder;
-    int64_t GeometryErrors;
-    int64_t TxBytes;
-    // Kafka stats below are common to all detectors
-    int64_t kafka_produce_fails;
-    int64_t kafka_ev_errors;
-    int64_t kafka_ev_others;
-    int64_t kafka_dr_errors;
-    int64_t kafka_dr_noerrors;
-  } __attribute__((aligned(64))) Counters;
+  // From Counters.h
+  struct Counters Counters;
```

Comment:
'struct' isnt necessary....

Reply:
I'm pretty sure we have compilers in the jenkins build system that complains if it is not there.
I have had this comment before. It builds fine on Mac but as far as I remember not on CentOS.